js/w3c/style.js

```diff
@@ -40,6 +40,9 @@ define(
                         .appendTo($("head"))
                         ;
                 }
+                else if (conf.useExperimentalStyles) {
+                    css = "http://fantasai.inkedblade.net/style/design/w3c-restyle/2016/W3C-" + statStyle;
```

Comment:
Well - that's where the stylesheets are.  Should we have done something
else?

On Wed, Nov 25, 2015 at 7:55 PM, Marcos Caceres <notifications@github.com>
wrote:

> In js/w3c/style.js
> <https://github.com/w3c/respec/pull/522#discussion_r45938038>:
>
> > @@ -40,6 +40,9 @@ define(
> >                          .appendTo($("head"))
> >                          ;
> >                  }
> > +                else if (conf.useExperimentalStyles) {
> > +                    css = "http://fantasai.inkedblade.net/style/design/w3c-restyle/2016/W3C-" + statStyle;
>
> D'oh. Can't believe I didn't see this :(
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/w3c/respec/pull/522/files#r45938038>.
>



-- 
Shane McCarron
halindrome@gmail.com